tests/e2e/functional_test.go

```diff
@@ -378,8 +378,10 @@ var _ = Describe("Functional e2e", Serial, func() {
 
 			document("Creating a slow pipeline")
 			slowPipelineSpec := updatedPipelineSpec.DeepCopy()
-			highRPU := int64(10000)
+			highRPU := int64(10000000)
```

Comment:
at first I was trying to just increase this before I realized the real issue was the pauseGracePeriodSeconds. But I just left this value here since it doesn't harm anything.